File: 9-palindrome-number/palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        if(x<0){
            return false;
        }
        int y=x;
        int t=0;
        while(x!=0){
            int d=x%10;
            t=10*t+d;
            x=x/10;
        }
        if(y==t){
            return true;
        }else{
      return false;  
        }
    }
}

File: 9-palindrome-number/palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        if (x<0):
            return False
        o=x
        t=0
        while(x!=0):
            d=x%10
            t=t*10+d
            x=x//10
        return o==t